adblock-detect-react/package.json

```diff
@@ -41,7 +41,7 @@
     "typescript": "^3.7.2"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^16.8.0 || ^17.0.0",
```

Comment:
Could you bump the patch version of the pkg to `1.0.6` ?

Reply:
done